### PR TITLE
Update get_acl_couner with command instead of getting value from redis

### PR DESCRIPTION
**Pull request by ZhaohuiS**
### Description of PR

Summary:
Fixes # (issue)
We can't get acl counters from redis COUNTERS_DB due to this RP https://github.com/Azure/sonic-swss/pull/1943.
Parse the output of `aclshow` to get counters.

Signed-off-by: Zhaohui Sun <zhaohuisun@microsoft.com>

### Type of change

- [x] Bug fix
- [ ] Testbed and Framework(new/improvement)
- [ ] Test case(new/improvement)

### Back port request
- [ ] 201911
- [ ] 202012

#### What is the motivation for this PR?
We can't get acl counters from redis `COUNTERS_DB` due to this RP https://github.com/Azure/sonic-swss/pull/1943.
`test_acl_outer_vlan.py` keeps failing  because of the error `Failed: Failed to retrieve acl counter for DATAACL_ingress_ipv6|rule_1`.

#### How did you do it?
Parse the output of `aclshow` to get counters.

#### How did you verify/test it?
run `acl/test_acl_outer_vlan.py`

### CODE REVIEW

File: tests/acl/test_acl_outer_vlan.py [hunk 1]
```diff
@@ -284,13 +284,16 @@ def get_acl_counter(duthost, table_name, rule_name, timeout=ACL_COUNTERS_UPDATE_
...
     # Wait for orchagent to update the ACL counters
     time.sleep(timeout)
-    result = duthost.shell(cmd.format(table_name, rule_name))['stdout']
-    if result == "":
-        pytest.fail("Failed to retrieve acl counter for {}|{}".format(table_name, rule_name))
-    return int(result)
+    result = duthost.show_and_parse('aclshow')
+
+    if len(result) == 0:
+        return 0
```
Comment (StormLiangMS) [inline tests/acl/test_acl_outer_vlan.py:292]: Is the "0" expected behavior? If not, could we some add debug/error print here?

File: tests/acl/test_acl_outer_vlan.py [hunk 2]
```diff
@@ -284,13 +284,16 @@ def get_acl_counter(duthost, table_name, rule_name, timeout=ACL_COUNTERS_UPDATE_
...
+    for rule in result:
+        if rule_name == rule['rule name']:
+            return int(rule['packets count'])
+    return 0
```
Comment (StormLiangMS) [inline tests/acl/test_acl_outer_vlan.py:296]: Same here

File: tests/acl/test_acl_outer_vlan.py [hunk 3]
```diff
@@ -284,13 +284,16 @@ def get_acl_counter(duthost, table_name, rule_name, timeout=ACL_COUNTERS_UPDATE_
...
         Acl counter value for packets
     """
-    cmd = "redis-cli -n 2 hget 'COUNTERS:{}:{}' Packets"
     # Wait for orchagent to update the ACL counters
     time.sleep(timeout)
-    result = duthost.shell(cmd.format(table_name, rule_name))['stdout']
-    if result == "":
-        pytest.fail("Failed to retrieve acl counter for {}|{}".format(table_name, rule_name))
-    return int(result)
+    result = duthost.show_and_parse('aclshow')
```
Comment (bingwang-ms) [inline tests/acl/test_acl_outer_vlan.py:289]: Do we need `aclshow -a`? I'm not sure if `aclshow` works. Please double check

File: tests/acl/test_acl_outer_vlan.py (hunk 1 above)
Comment (ZhaohuiS) [reply tests/acl/test_acl_outer_vlan.py:292]: @StormLiangMS If there is no rule for acl, the output looks like below,  result is empty.
We return 0 here. The log file includes all the output of shell command.
```
admin@str-dx010-acs-5:~$ aclshow -a
RULE NAME    TABLE NAME    PRIO    PACKETS COUNT    BYTES COUNT
-----------  ------------  ------  ---------------  -------------
```

Comment (ZhaohuiS) [reply tests/acl/test_acl_outer_vlan.py]: @StormLiangMS Rethinking about your comment, you are correct. get_acl_counter is called after setup acl rules, so if can't find any expected rule, fail the test case.

File: tests/acl/test_acl_outer_vlan.py (hunk 3 above)
Comment (ZhaohuiS) [reply tests/acl/test_acl_outer_vlan.py]: Done

File: tests/acl/test_acl_outer_vlan.py (hunk 2 above)
Comment (ZhaohuiS) [reply tests/acl/test_acl_outer_vlan.py]: It's done in my second commit.

**Review by bingwang-ms — APPROVED** [review]

**Pull request merged**